src/tokens/solana.tokenlist.json

```diff
@@ -6399,11 +6399,23 @@
       "extensions": {
         "website": "https://sola.farm/"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "DzWgNKrSoTHQQXZvBhcpsQ7EMrHeTVuRM6LAm8Cwr1m4",
+      "symbol": "LUCK",
+      "name": "Lucky",
+      "decimals": 0,
+      "logoURI": "http://www.lucky-token.online/images/logo1.png",
```

Comment:
can you use https for hosting resources.

Reply:
Hi! I've made the changes required. 

Thank you very much